### PR TITLE
use our own build.rs helper

### PR DESCRIPTION
**Pull request by Wescoeur**

### CODE REVIEW

File: rust/build.rs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// See: https://rust-lang.github.io/rust-bindgen/tutorial-3.html
+use std::{env, path::PathBuf};
+
+const WRAPPER: &str = "lib/wrapper.h";
+
+fn main () {
+  println!("cargo:rerun-if-changed={}", WRAPPER);
+  println!("cargo:rustc-link-lib={}={}", "dylib", "xenctrl");
```
Comment (benjamreis) [inline rust/build.rs:8]: why not linking against xenstore as well?

File: rust/lib/lib.rs [hunk 2]
```diff
@@ -1,3 +1,5 @@
 pub mod vm;
 pub mod xenctrl;
 pub mod xenstore;
+
+mod bindings;
```
Comment (benjamreis) [inline rust/lib/lib.rs:5]: Is it possible to have 2 namespaces?
`bindings::xenctrl` & `bindings::xenstore`

For readability purpose.

Comment (Wescoeur) [reply rust/lib/lib.rs:5]: It's possible. But each function in `xenctrl` and `xenstore` has a prefix like`xs_` or `xc`. It's not very useful to add a namespace.

File: rust/build.rs (hunk 1 above)
Comment (Wescoeur) [reply rust/build.rs:8]: Because I forgot that. :slightly_smiling_face:

**Review by benjamreis — APPROVED** [review]

**Pull request merged**